### DISCUSSION TITLE
Add "Gameplay cursor size" setting in the loading screen under "visual settings"

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#30457] [Ideas]
I like to change my cursor size manually depending on the circle size and instead of always going into the settings tab, it would be more convenient to change it on the loading screen. 
![png](https://github.com/user-attachments/assets/0b30297c-3f62-4d91-b706-0d12bf98872f)

**Comment by Joehuu** [comment]
Does the "adjust gameplay cursor size based on current beatmap" not work for you? It automatically scales the cursor based on circle size.

**Reply by ghost** [reply]
I don't like the option because, at higher cs values, cursors get too small for my taste. I prefer using constant size values depending on whether I play with nm, hr, or whether the cs is above 7

**Reply by cl8n** [reply]
imo it's just too specific of a personal issue to warrant cluttering that menu at the start of every map with an option most people would ignore.

maybe some kind of "favorite settings" thing where you can choose options to pin here would work?
